core/src/epicli/data/common/ansible/playbooks/roles/applications/templates/ignite-stateless/04-deployment.yml.j2

```diff
@@ -22,7 +22,11 @@ spec:
       containers:
         # Custom Ignite pod name.
       - name: ignite-node
+{% if data.use_local_image_registry is undefined or data.use_local_image_registry is sameas true %}
```

Comment:
`data.use_local_image_registry is sameas true` - may it be shorter? There are also other similar changes in this PR.

Reply:
It should be `data.use_local_image_registry | bool` actually this time :thinking: I just copy/pasted it from existing code. Let me work on it :+1:

Reply:
Updated in the latest commit.